spec/features/profiles_spec.rb

```diff
@@ -74,3 +74,36 @@
     end
   end
 end
+
+
+feature 'Websites New' do
+  describe "Form" do
+    before :each do
+      Website.create(
+        name: 'Sample Website',
+        url: 'http://www.sample.com/'
+      )
+      visit '/profiles/new'
+    end
+
+    it "has forms" do
+      expect(page).to have_selector('input#profile_name')
+      expect(page).to have_selector('select#profile_website_id')
+      expect(page).to have_selector('input#profile_url')
+    end
+    it "can not submit form without required fields" do
+      click_on('Add New Profile')
+      expect(page.current_path).to eql new_profile_path
+    end
+    it "has error responses"
+    it "can submit form" do
+      fill_in('Name', :with => 'Sample Profile')
+      select('Sample Website', :from => 'Website')
+      fill_in('Url', :with => 'http://www.sample.com')
+      click_on('Add New Profile')
+      expect(page.current_path).to eql profiles_path
+      expect(page).to have_content("Sample Profile")
```

Comment:
generally, it's good practice to only have one expect per spec.  This way it's easier to debug when a spec fails because there's no hunting down which expectation line failed inside your spec.